Partnermanagement/PEX-Partner-API.md

```diff
@@ -289,6 +290,94 @@ Content-Type: application/json;charset=utf-8
 }
 ```
 
+## Details zu einem Partner
+
+Bitte nutzen sie die Relationen des Knotens `_links` der Partner-Entität.
+Es handelt sich hierbei um ein älteres API-Konzept, weshalb künftige Änderungen wahrscheinlich sind.
+Wir beabsichtigen die Clientkompatibilität über die Relationen sicherzustellen.
+Über die Bereitstellung einer neueren API werden wir informieren.
+
+### Partnerkennzeichen
+Die Partnerkennzeichen identifizieren einen Vertrieb bei einem Produktanbieter.
+Die erfassten Werte können über diesen Endpunkt ausgelesen werden.
+
+#### GET Request Beispiele
+
+```
+GET https://www.europace2.de/partnermanagement/partner/AB123/partnerkennzeichen
+X-ApiKey: xxxxxx
+X-PartnerId: AB123
+X-TraceId: request-2014-10-01-07-59
+Accept: application/json
+```
+
+#### GET Response Beispiel:
+
+```
+200 OK
+X-TraceId: request-2014-10-01-07-59
+Content-Type: application/json;charset=utf-8
+
+{
+    "dslSapGeschaeftspartnerNummerFuerRatenkredit": "eineNummer",
```

Comment:
Hier vielleicht die echten Partnerkennzeichen rausnehmen und mit 

> "kennzeichenBeispiel": "EIN_PKZ" 

arbeiten

Reply:
In Absprache mit SMo können wir die Struktur so dokumentieren. Ich habe aber noch fehler im json entdeckt `"null"`, den werde ich noch beseitigen.